BiweeklyMeetings/2022-05-24.md

```diff
@@ -0,0 +1,563 @@
+
+**Matthias Benkort**: Yes, I think both what you said.
+
+**M. Àngels Jover**:  Okay. Now I wanted to know how you do that. I guess that is like... As you are in control to changing the proposals from proposed to active. So this would be like a kind of requirement before moving it to active. And this would be in a standby until we have this simulation.
+
+**Matthias Benkort**: Yeah, indeed active, to active or rejected depending on the outcome of the results.
+
+**M. Àngels Jover**:  Okay. [inaudible 00:32:04].
+
+**Matthias Benkort**: If the simulation clearly shows that the proposal isn't sound, that it would make the situation worse, then we just don't want to make... Move it as active, but just as rejected.
+
+### Last Check 
+
+### Review 
+
+#### PR242
+ [PR242 CIP-0050? | Shelleys Voltaire decentralization update](https://github.com/cardano-foundation/CIPs/pull/242)
+
+**M. Àngels Jover**:  Okay. So, that were the items we had in triage. And now in review, we have PR242, which is CIP 50 regarding Shelleys Voltaire Decentralization Update. I think that we are going to do a weekly summary on this, if I am correct?
+
+**Matthias Benkort**: Yes. So I actually spent the whole day looking at the proposal. I'm going through a thread of comments. So I try to provide a summary on the bottom. There has been many, many things been said, but it's mostly also a debate around whether or not the proposal improves the situations in terms of, does it benefit decentralization, and does it benefit actually every users of the ecosystem and not just the big players?
+
+**Matthias Benkort**: So the discussion hasn't really settled on that. And again, this is kind of proposal where we need more evidence or stronger arguments in favor or defavor of the proposal, and something, hopefully this new simulator from the research team could help with. The research team is also aware of that proposal. I've had a discussion with IOG also on this one. He also mentioned that it was quite close to how he works and incentives works on Tezos currently, and that it has improved.
+
+**Matthias Benkort**: They had analyzed in the past, and that they had rejected or iterated upon in the current design that is used in Cardano. He pointed out a blog post he wrote two years ago on the topic, which we'll go over the different designs that you can use for incentive and rewards, and why this proposed design has issues and works actually worse than the current mechanism. Which is also one point being made during... By other people in this whole thread of comments, which is that it's also not clearly established that the current mechanism is worse than the proposal here. And that possibly with a different set of parameters, we could also alleviate some of the issues that are being brought out or motivating this [inaudible 00:35:10].So that's a bit from the summary of the discussions on that.
```

Comment:
```suggestion
**Matthias Benkort**: They had analyzed in the past, and that they had rejected or iterated upon in the current design that is used in Cardano. He pointed out a blog post he wrote two years ago on the topic, which goes over the different designs that you can use for incentive and rewards, and why this proposed design has issues and works actually worse than the current mechanism. Which is also one point being made during... By other people in this whole thread of comments, which is that it's also not clearly established that the current mechanism is worse than the proposal here. And that possibly with a different set of parameters, we could also alleviate some of the issues that are being brought out or motivating this proposal. So that's a bit from the summary of the discussions on that.
```